Feature Guides/Sensitivity_Analysis.md

```diff
@@ -0,0 +1,173 @@
+# LDAR-Sim Sensitivity Analysis Guide
+
+## Introduction
+
+The purpose of this guide is to provide users with instructions and examples on how to run the LDAR-Sim Sensitivity Analysis module. Sensitivity Analysis with the LDAR-Sim.
```

Comment:
Is this supposed to say "The purpose of this guide is to provide users with instructions and examples on how to run a sensitivity analysis with the LDAR-Sim Sensitivity Analysis module." ?

Reply:
Great catch I fixed up the wording